src/sensor/ping.cpp

```diff
@@ -195,15 +198,17 @@ void Ping::handleMessage(PingMessage msg)
 
         // This is necessary to convert <uint8_t> to <int>
         // QProperty only supports vector<int>, otherwise, we could use memcpy
-        for (int i = 0; i < m.num_points(); i++) {
-            _points.replace(i, m.data()[i] / 255.0); // TODO we should really be working in ints
+        qCDebug(PING_PROTOCOL_PING) << "length:" << m.profile_data_length();
```

Comment:
For high frequency messages, we should not use QCDebug. Right now all messages does have the same size.

Reply:
whoops, should not be here.

Reply:
After doing some tests, I think that this message can be removed. This is almost half of the debug messages and does not help to debug, the profile has a fixed length, and if fails the checksum will show it to us.
```
[19:42:15:584] ping.protocol.ping[Debug]: ping.cpp(246) length: 200
[19:42:15:592] ping.protocol.ping[Debug]: ping.cpp(149) Handling Message: 1300 Checksum Pass: true
[19:42:15:593] ping.protocol.ping[Debug]: ping.cpp(246) length: 200
[19:42:15:665] ping.protocol.ping[Debug]: ping.cpp(149) Handling Message: 1300 Checksum Pass: true
[19:42:15:676] ping.protocol.ping[Debug]: ping.cpp(246) length: 200
[19:42:15:693] ping.protocol.ping[Debug]: ping.cpp(149) Handling Message: 1300 Checksum Pass: true
[19:42:15:702] ping.protocol.ping[Debug]: ping.cpp(246) length: 200
[19:42:15:727] ping.protocol.ping[Debug]: ping.cpp(149) Handling Message: 1300 Checksum Pass: true
[19:42:15:728] ping.protocol.ping[Debug]: ping.cpp(246) length: 200
[19:42:15:766] ping.protocol.ping[Debug]: ping.cpp(149) Handling Message: 1300 Checksum Pass: true
[19:42:15:767] ping.protocol.ping[Debug]: ping.cpp(246) length: 200
[19:42:15:799] ping.protocol.ping[Debug]: ping.cpp(149) Handling Message: 1300 Checksum Pass: true
[19:42:15:801] ping.protocol.ping[Debug]: ping.cpp(246) length: 200
[19:42:15:891] ping.protocol.ping[Debug]: ping.cpp(149) Handling Message: 1300 Checksum Pass: true
[19:42:15:892] ping.protocol.ping[Debug]: ping.cpp(246) length: 200
[19:42:15:939] ping.protocol.ping[Debug]: ping.cpp(149) Handling Message: 1300 Checksum Pass: true
[19:42:15:940] ping.protocol.ping[Debug]: ping.cpp(246) length: 200
[19:42:16:000] ping.protocol.ping[Debug]: ping.cpp(149) Handling Message: 1300 Checksum Pass: true
[19:42:16:001] ping.protocol.ping[Debug]: ping.cpp(246) length: 200
[19:42:16:016] ping.protocol.ping[Debug]: ping.cpp(149) Handling Message: 1300 Checksum Pass: true
[19:42:16:028] ping.protocol.ping[Debug]: ping.cpp(246) length: 200
[19:42:16:157] ping.protocol.ping[Debug]: ping.cpp(149) Handling Message: 1300 Checksum Pass: true
[19:42:16:166] ping.protocol.ping[Debug]: ping.cpp(246) length: 200
[19:42:16:216] ping.protocol.ping[Debug]: ping.cpp(149) Handling Message: 1300 Checksum Pass: true
[19:42:16:217] ping.protocol.ping[Debug]: ping.cpp(246) length: 200
[19:42:16:224] ping.protocol.ping[Debug]: ping.cpp(149) Handling Message: 1300 Checksum Pass: true
[19:42:16:225] ping.protocol.ping[Debug]: ping.cpp(246) length: 200
[19:42:16:244] ping.protocol.ping[Debug]: ping.cpp(149) Handling Message: 1300 Checksum Pass: true
[19:42:16:245] ping.protocol.ping[Debug]: ping.cpp(246) length: 200
[19:42:17:988] ping.protocol.ping[Debug]: ping.cpp(149) Handling Message: 1300 Checksum Pass: true
[19:42:17:989] ping.protocol.ping[Debug]: ping.cpp(246) length: 200
[19:42:19:280] ping.protocol.ping[Debug]: ping.cpp(149) Handling Message: 1300 Checksum Pass: true
[19:42:19:282] ping.protocol.ping[Debug]: ping.cpp(246) length: 200
[19:42:20:830] ping.protocol.ping[Debug]: ping.cpp(149) Handling Message: 1300 Checksum Pass: true
[19:42:20:842] ping.protocol.ping[Debug]: ping.cpp(246) length: 200
[19:42:22:567] ping.protocol.ping[Debug]: ping.cpp(149) Handling Message: 1300 Checksum Pass: true
[19:42:22:568] ping.protocol.ping[Debug]: ping.cpp(246) length: 200
[19:42:24:748] ping.protocol.ping[Debug]: ping.cpp(149) Handling Message: 1300 Checksum Pass: true
[19:42:24:749] ping.protocol.ping[Debug]: ping.cpp(246) length: 200
[19:42:26:517] ping.protocol.ping[Debug]: ping.cpp(149) Handling Message: 1300 Checksum Pass: true
[19:42:26:518] ping.protocol.ping[Debug]: ping.cpp(246) length: 200
[19:42:28:496] ping.protocol.ping[Debug]: ping.cpp(149) Handling Message: 1300 Checksum Pass: true
[19:42:28:497] ping.protocol.ping[Debug]: ping.cpp(246) length: 200
[19:42:30:146] ping.protocol.ping[Debug]: ping.cpp(149) Handling Message: 1300 Checksum Pass: true
[19:42:30:149] ping.protocol.ping[Debug]: ping.cpp(246) length: 200
[19:42:30:157] ping.protocol.ping[Debug]: ping.cpp(149) Handling Message: 1300 Checksum Pass: true
[19:42:30:159] ping.protocol.ping[Debug]: ping.cpp(246) length: 200
[19:42:30:172] ping.protocol.ping[Debug]: ping.cpp(149) Handling Message: 1300 Checksum Pass: true
[19:42:30:174] ping.protocol.ping[Debug]: ping.cpp(246) length: 200
[19:42:30:189] ping.protocol.ping[Debug]: ping.cpp(149) Handling Message: 1300 Checksum Pass: true
[19:42:30:191] ping.protocol.ping[Debug]: ping.cpp(246) length: 200
[19:42:30:201] ping.protocol.ping[Debug]: ping.cpp(149) Handling Message: 1300 Checksum Pass: true
[19:42:30:212] ping.protocol.ping[Debug]: ping.cpp(246) length: 200
[19:42:30:218] ping.protocol.ping[Debug]: ping.cpp(149) Handling Message: 1300 Checksum Pass: true
[19:42:30:219] ping.protocol.ping[Debug]: ping.cpp(246) length: 200
[19:42:30:237] ping.protocol.ping[Debug]: ping.cpp(149) Handling Message: 1300 Checksum Pass: true
[19:42:30:238] ping.protocol.ping[Debug]: ping.cpp(246) length: 200
[19:42:30:253] ping.protocol.ping[Debug]: ping.cpp(149) Handling Message: 1300 Checksum Pass: true
[19:42:30:254] ping.protocol.ping[Debug]: ping.cpp(246) length: 200
[19:42:30:288] ping.protocol.ping[Debug]: ping.cpp(149) Handling Message: 1300 Checksum Pass: true
[19:42:30:289] ping.protocol.ping[Debug]: ping.cpp(246) length: 200
[19:42:30:296] ping.protocol.ping[Debug]: ping.cpp(149) Handling Message: 1300 Checksum Pass: true
[19:42:30:302] ping.protocol.ping[Debug]: ping.cpp(246) length: 200
```

Reply:
You are so right. I'm sorry to leave it here by mistake.